src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortTest.java

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+
+class ExtractCohortTest extends CommandLineProgramTest {
+  private final String prefix = getToolTestDataDir();
+  private final String cohortAvroFileName = prefix +"chr20_subset_3_samples.avro";
+  private final String sampleFile = prefix +"sample_list";
+  private final String outputFileName = prefix +"output.vcf";
+
+  @Test
+  public void testFinalVCFfromAvro() throws Exception {
+    // To create the expected output file--create a temp table in BQ with the folowing query
+    // and then export it through the GUI as an avro file into GCS.
+    // SELECT * FROM `spec-ops-aou.anvil_100_for_testing.exported_cohort_all_samples`
+    // where location < 20000000200000 and location >= 20000000100000
+    // and (sample_name="HG00405" or sample_name="HG00418" or sample_name="HG00408")
+
+    final ArgumentsBuilder args = new ArgumentsBuilder();
+    args
+        .add("mode", "GENOMES")
+        .add("ref-version", 38)
+        .add("query-mode", "LOCAL_SORT")
+        .add("R", hg38Reference)
+        .add("O", outputFileName)
+        .add("local-sort-max-records-in-ram", 10000000)
+        .add("cohort-avro-file-name", cohortAvroFileName)
+        .add("sample-file", sampleFile);
+
+    runCommandLine(args);
+
+    final File expectedVCF = getTestFile("expected.vcf");
+    final File outputVCF = getTestFile("output.vcf");
```

Comment:
Is this going to get the `output.vcf` that you checked in as part of this commit?  It should be the output of the run above (which I think won't live in the same place as getTestFile().  You can tell for sure by removing `src/test/resources/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohort/output.vcf` from your PR (which I think shouldn't be commited anyway).